### ISSUE TITLE
Title does not show

### ISSUE THREAD

**Issue by WolfgangFahl** [HanSolo/Medusa#120]
In the Overview the Gauge: 
```
gauge20 = GaugeBuilder.create()
                              .skinType(SkinType.LEVEL)
                              .title("Capacity")
                              .titleColor(Color.WHITE)
                              .animated(true)
                              .gradientBarEnabled(true)
                              .gradientBarStops(new Stop(0.0, Color.RED),
                                                new Stop(0.25, Color.ORANGE),
                                                new Stop(0.5, Color.YELLOW),
                                                new Stop(0.75, Color.YELLOWGREEN),
                                                new Stop(1.0, Color.LIME))
                              .build();
```
does not show a title and the titleVisible is not available.

**Comment by WolfgangFahl** [comment]
also it does not react on changing the decimals ...

**Comment by HanSolo** [comment]
Fixed with commit 9202239b160d1d64b1a364d27e1dd26a31d8ec33
Please also keep in mind that not all Skins support all properties :)
